common/components/timeline/macro.njk

```diff
@@ -0,0 +1,27 @@
+{% macro appTimeline(params) %}
+  {%- include "./template.njk" -%}
+{% endmacro %}
+
+{% macro appTimelineItemHeading(params, item) %}
+  {%- include "./timeline--heading.njk" -%}
+{% endmacro %}
+
+{% macro appTimelineItemDate(params, item) %}
+  {%- include "./timeline--date.njk" -%}
+{% endmacro %}
+
+{% macro appTimelineItemDescription(params, item) %}
+  {%- include "./timeline--description.njk" -%}
+{% endmacro %}
+
+{% macro appTimelineItemAdditional(params, item) %}
+  {%- include "./timeline--additional.njk" -%}
+{% endmacro %}
+
+{% macro appTimelineItem(params, item) %}
+{%- include "./timeline--item.njk" -%}
+{% endmacro %}
+
+{% macro appTimelineItemAlt(params, item) %}
```

Comment:
@solidgoldpig I had a quick conversation with @Andy-Millar to help me better understand these changes.

It sounds like he feels we will **not** use the proposed format from the MOJ Design system. So in that vein I would suggest we don't need to extract all of this logic in order to support both and just build this component in the format that we have decided.

That would simplify what it is we need to build at this stage.

Reply:
As previous comment